.github/ISSUE_TEMPLATE/request_cluster_resources.md

```diff
@@ -0,0 +1,31 @@
+---
+name: General cluster-scoped resource request
+about: "Requesting resources which require cluster-admin: CRDs, ClusterRoles, Operators, Quotas"
+title: ""
+labels: onboarding
+assignees: ""
+---
+
+### Requested actions
+
+#### Operator request
+
+If you want to request an operator to be deployed either with all namespaces visibility or with a single namespace visibility, please state here which operator you are interested in. Please point us to any documentation about this operator directly
```

Comment:
I think we can be a bit more forward here, suggestion: 

>If you want to request an operator, please provide the following: 
>- Name of the Operator
>- Operator version
>- Should the Operator be visible to a _Single Namespace_ or _All Namespaces_ 
>- Any documentations about this operator (if applicable)

Reply:
Applied, thanks :+1: